### PR TITLE
Fix canonicalization

### PR DESCRIPTION
**Pull request by karlbaker02**
This PR fixes canonicalization with the `rsfcreator.py` script so that empty key-value pairs are not included in metadata items and to stop escaping special characters as unicode.

### CODE REVIEW

File: scripts/rsfcreator.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def rsf_for_line(line_dict, key_field, entry_type, key_prefix='', key=None):
     if key is None:
         key = line_dict[key_field]
     timestamp = datetime.utcnow().strftime('%Y-%m-%dT%H:%M:%SZ')
-    item_str = json.dumps(line_dict, separators=(',', ':'), sort_keys=True)
+    item_str = json.dumps(line_dict, ensure_ascii=False, separators=(',', ':'), sort_keys=True)
```
Comment (gidsg) [inline scripts/rsfcreator.py]: Are these fixes testable, are they worth testing?

Comment (michaelabenyohai) [inline scripts/rsfcreator.py]: Good point. We do have a test class for `rsfcreator.py`. Can we add some tests @karlbaker02 ?

Comment (karlbaker02) [inline scripts/rsfcreator.py]: Yes, I'll do this, thanks.

Comment (karlbaker02) [inline scripts/rsfcreator.py]: Test has now been added.

**Pull request merged**